applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.cpp

```diff
@@ -780,6 +639,19 @@ unsigned int UPwBaseElement<TDim, TNumNodes>::GetNumberOfDOF() const
     return TNumNodes * (TDim + 1);
 }
 
+template <unsigned int TDim, unsigned int TNumNodes>
+Element::DofsVectorType UPwBaseElement<TDim, TNumNodes>::GetDofs() const
```

Comment:
This will probably be the only remaining implementation for the UPw element when the whole redesign is complete.

Reply:
Yes, that makes sense.